prelude/size.md

```diff
@@ -30,8 +28,16 @@ impl Size {
         Size { raw }
     }
 
+    /// variation of `from_bits` for const contexts.
+    /// Cannot fail since the input is unsigned.
+    pub const fn from_bits_const(bits: u64) -> Size {
+        let bytes = bits / 8 + ((bits % 8) + 7) / 8;
```

Comment:
This is not exactly readable, we should find a way to express this more nicely...

In fact [div_ceil](https://doc.rust-lang.org/nightly/std/primitive.u64.html#method.div_ceil) seems like exactly the right thing? Can you use that here, and also add it to `Int`? I'd like to keep MiniRust clean of terrible things like "how to do integer division rounding up"^^